README.markdown

```diff
@@ -277,6 +277,30 @@ Tells Puppet what NTP service to manage. Valid options: string. Default value: v
 
 Tells puppet to change stepout. Applies only if `tinker` value is 'true'. Valid options: unsigned shortint digit. Default value: undef.
 
+####`tos`
+
+Tells Puppet to enable tos options. Valid options: 'true' of 'false'. Default value: 'false'
+
+####`tos_minclock`
+
+Specifies the minclock tos option. Valid options: numeric. Default value: varies by operating system
+
+####`tos_minsane`
+
+Specifies the minsane tos option. Valid options: numeric. Default value: varies by operating system
+
+####`tos_floor`
+
+Specifies the floor tos option. Valid options: numeric. Default value: varies by operating system
+
+####`tos_ceiling`
+
+Specifies the ceiling tos option. Valid options: numeric. Default value: varies by operating system
+
+####`tos_cohort`
+
+Specifies the cohort tos option. Valid options: '0' or '1'. Default value: varies by operating system
+
```

Comment:
For all of these `Default value: varies by operating system`, it looks like params actually has a consistent default for all oses, so that should probably be listed instead.